### PR TITLE
Add Jenkinsfile

### PR DESCRIPTION
**Pull request by ac-miller**
## what
* Add Jenkinsfile for Jenkins pipeline
* Removed duplicated directories in src/main and src/test
* Added tasks to Taskfile

## why
* We need a Jenkinsfile to run a pipeline automatically when EVTR starts
* We had several duplicated directories that came from renaming the project

### CODE REVIEW

**Review by jesse-chauvin — DISMISSED** [review]
Approved

**Review by Colton-Wrisner — DISMISSED** [review]

**Review by jesse-chauvin — DISMISSED** [review]
Changes are good.

**Comment by RothAndrew** [comment]
Every container can't have that much cpu/memory, the pod will never get scheduled. With this config the pod needs 6 cpu and 12 GB of memory

**Comment by ac-miller** [comment]
The pod does get scheduled through testing, but I see what you mean. I'm going to test with the jnlp and dind containers requesting/limiting minimal cpu and memory.

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+      resources:
+        requests:
+          cpu: "250m"
+          memory: "512Mi"
+        limits:
+          cpu: "250m"
+          memory: "512Mi"
+    - name: anvil
+      image: saicoss/anvil:0.5.4
+      imagePullPolicy: Always
```
Comment (RothAndrew) [inline Jenkinsfile:23]: Change these to `IfNotPresent`

File: Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+  agent {
+    kubernetes {
+      //cloud 'kubernetes'
+      label "venus-demo-${UUID.randomUUID().toString()}"
+      yaml """
+apiVersion: v1
+kind: Pod
+spec:
+  imagePullSecrets:
+    - name: gitlab-registry-access
```
Comment (RothAndrew) [inline Jenkinsfile:11]: This can be removed

**Review by RothAndrew — CHANGES_REQUESTED** [review]

**Review by RothAndrew — APPROVED** [review]

**Pull request merged**